src/platform/macos.rs

```diff
@@ -186,6 +179,20 @@ impl WindowBuilderExtMacOS for WindowBuilder {
     }
 }
 
+pub trait EventLoopExtMacOS {
+    /// Sets the activation policy for the application.
+    /// It is set to `NSApplicationActivationPolicyRegular` by default.
+    fn set_activation_policy(&mut self, activation_policy: ActivationPolicy);
```

Comment:
It should be documented that this can only be called before `EventLoop::run`.

Or, alternatively, it should be as a `with_activation_policy_and_user_event` creation method (and in the future, if the pattern is common enough, we could create `EventLoopBuilder`)

Reply:
Will do!